pom.xml

```diff
@@ -437,7 +432,6 @@
             </goals>
             <configuration>
               <ignoreNonCompile>true</ignoreNonCompile>
-              <failOnWarning>true</failOnWarning>
```

Comment:
we had pervious issue with this settings, and ended mismatch dependencies, leading to error. So removing this flag